main.go

```diff
@@ -61,6 +61,11 @@ var flags = []cli.Flag{
 		Usage: "number of expected peers in the cluster",
 		Value: 2,
 	},
+	&cli.IntFlag{ // TODO: Probably remove this once done debugging
```

Comment:
Consider making this a `BoolFlag`, which you can set to `true` just by passing `-foo` with no arguments.